__TESTS__/text-diff.spec.tsx

```diff
@@ -27,4 +27,35 @@ describe('TextDiff', () => {
 
     expect(output.innerHTML).toBe(testOutput);
   });
+
+  it('Show different characters ignoring case', async () => {
+    const randomCase = (str: string): string =>
+      str
+        .split('')
+        .map((character) =>
+          Math.round(Math.random()) === 0
+            ? character.toLowerCase()
+            : character.toUpperCase(),
+        )
+        .join();
+
+    userEvent.selectOptions(
+      screen.getByLabelText(/Choose Diff Options/i),
+      ['Characters Ignore Case'],
+    );
+
+    const user = userEvent.setup();
+    render(<TextDiff />);
```

Comment:
You have to render the component before you can query against it.

I generally try to keep the user event setup and render at the top of each test case. Give that a shot and it should work for you.

Reply:
Yea that worked perfectly. Sorry, it was such a dumb mistake. Thanks a lot!

Reply:
No worries! We all miss things like that from time to time!